docs/download_options.md

```diff
@@ -39,14 +39,14 @@ For more information about the expected file download structure for "Single-cell
 
 ## Merge options
 
-When downloading a project, either by using `Download Now` or `Add to Dataset`, you will have the option to either receive the data as objects for individual libraries, or as {ref}`a single merged object with data from all samples in the given project<merged_objects.md:Merged objects>`.
+When downloading a project, either by using `Download Now` or `Add to Dataset`, you will have the option to either receive the data as objects for individual libraries, or as {ref}`a single merged object with data from all samples in the given project<merged_objects:Merged objects>`.
 Please be aware that merged objects have _not_ been integrated or batch-corrected.
 Refer to {ref}`this documentation<download_files:Merged object downloads` for the contents of a merged object download specifically.
 Note that this applies only to "Single-cell" modality downloads, not "Spatial."
 
 When {ref}`creating a custom dataset to download<download_files:Custom datasets>`, you will be able to select the option to merge all samples only if you have included all samples from the given project in `My dataset`.
 Merging a subset of samples in a project {ref}`is not currently supported<faq:STUB for https://github.com/AlexsLemonade/scpca-docs/issues/399>`.
-In addition, merged objects are not available for all samples or projects, {ref}`as described here<faq:Which projects can I download as merged objects?`.
+In addition, merged objects are not available for all samples or projects, {ref}`as described here <faq:Which projects can I download as merged objects?>`.
```

Comment:
I don't actually think we want this space? Do we?

Reply:
Looking at how we format other refs, we have some references with spaces and some without. The bigger change here is the `>` at the end, but I don't think the space matters.

Reply:
Ah I hadn't noticed the `>`. These refs....

Reply:
The rendered version here is ok with this space, so we can leave it.